recipes/go/build.sh

```diff
@@ -0,0 +1,24 @@
+# First, build go1.4 using gcc, then use that go to build go>1.4
+# Should I make this a separate package called go-bootstrap?
+# Should I check the checksum of this tarball? What about windows?
+mkdir go-bootstrap && pushd $_
+curl -L https://storage.googleapis.com/golang/go1.4-bootstrap-20170531.tar.gz | tar -xzf -
+export GOROOT_BOOTSTRAP=$PWD/go
+cd $GOROOT_BOOTSTRAP/src
+./make.bash
+
+# Should I run all.bash instead? That will run tests too, but some tests
+# like unshare() require higher capabilities which might not be available
+# on the CI systems
+pushd $SRC_DIR/src
+./make.bash
```

Comment:
IIRC `all.bash` does work and runs relatively quickly surprisingly. Though I do recall some issues on Travis CI. That said, I think I may have encountered those issues offline too. It has been a while so my memory is a little rough.

Reply:
I have added patches now for the Linux build to pass on CircleCI. The Travis CI ones are weird. They pass locally on my 10.9.5 macOS machine.